### DISCUSSION TITLE
Zustand store does not trigger rerender in CI

### DISCUSSION THREAD

**Discussion by jmsims2** [pmndrs/zustand#1488] [Q&A]
We have an app that is being converted from angularjs to react in place. We have a zustand store that shares data from angularjs to various react microfrontends. Everything works great in real life. However, when running end to end tests in playwright, many times the store won't trigger a rerender when data is changed and for the life of me I cannot figure out why. Our usage of zustand is essentially what's in the docs. Nothing customized. Has anyone run into issues in CI before?

**Comment by coskunuyar** [comment]
I think you need to create singleton of zustand node module then all mfes can use same instance of zustand
